app/ui/react-day-picker.css

```
@@ -3,9 +3,9 @@
 .rdp-root {
   --rdp-accent-color: var(--accent); /* The accent color used for selected days and UI elements. */
   --rdp-accent-background-color: #fafafa; /* The accent background color used for selected days and UI elements. */
-  --rdp-font-family: Chicago; /* The font family used by the calendar. Note that `inherit`does not work here. */
+  --rdp-font-family: Czikago; /* The font family used by the calendar. Note that `inherit`does not work here. */
```

Stylelint (17.13.0)

[error] Expected "Czikago" to be "czikago" (value-keyword-case)

(value-keyword-case)